WordPress/Classes/Models/Blog.m

```diff
@@ -733,6 +735,11 @@ - (BOOL)supportsVideoPress
     return self.isHostedAtWPcom;
 }
 
+- (BOOL)supportsVideoPressV5
+{
+    return self.isHostedAtWPcom || self.isAtomic || [self hasRequiredJetpackVersion:@"8.5"];
```

Comment:
See related comment on Android PR for selecting this version number: https://github.com/wordpress-mobile/WordPress-Android/pull/20181/files#r1497916911